extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxImagesHelper.php

```diff
@@ -63,20 +63,16 @@ public function extendImageData( $data, $thumbnailFileWidth, $thumbnailImgTagWid
 
 		return array_merge( $data, [
 			'ref' => $ref,
-			'height' => $imgTagDimensions['height'],
-			'width' => $imgTagDimensions['width'],
+			'height' => intval( $imgTagDimensions['height'] ),
```

Comment:
are we doing some math operations on it?
if not, I'd send it as string

Reply:
I did it because it started to return float values. It is used only as width and height attributes in html